projects/core/src/config/utils/dynamic-template.ts

```diff
@@ -0,0 +1,14 @@
+export function dynamicTemplate(
```

Comment:
What would be the reason to not inject this in the DI as a service?

Reply:
Because we are doing old school JS. No but really I didn't think of it, will do.